packages/react/src/ActionList/Item.tsx

```diff
@@ -112,7 +120,12 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
 
     const itemSelectionAttribute = selectionAttribute || inferredSelectionAttribute
     // Ensures ActionList.Item retains list item semantics if a valid ARIA role is applied, or if item is inactive
-    const listSemantics = listRole === 'listbox' || listRole === 'menu' || inactive || container === 'NavList'
+    const listItemSemantics =
+      role === 'option' || role === 'menuitem' || role === 'menuitemradio' || role === 'menuitemcheckbox'
+
+    const listRoleTypes = ['listbox', 'menu', 'list']
```

Comment:
Oooh, `list` is a new addition here. Can you say a bit more about why `list` was added here and what the remaining cases when we want buttonSemantics to be used?

Reply:
We don't have any live cases of `role="list"` with `ActionList`, so this was mainly just an escape hatch. There are odd cases out there where the new semantics might not be as ideal, one I can think of is when ActionList is utilized as a way to load in items (e.g. [loading items story](https://ui.githubapp.com/storybook/?path=/story/others-listviewitemsissuesprs--list-loading-example)). The only issue is, they probably shouldn't be interactive if they're `role="list"`. 

Not really beholden to having it here, but thought it could be useful as a escape hatch for components that still need the old semantics, at least temporarily.